.vscode/launch.json

```
@@ -108,6 +108,10 @@
       "type": "extensionHost",
       "request": "launch",
       "args": ["--extensionDevelopmentPath=${workspaceFolder}/packages/typespec-vscode"],
+      "outFiles": [
+        "${workspaceFolder}/packages/*/dist/**/*.cjs",
+        "${workspaceFolder}/packages/*/dist-dev/**/*.cjs"
+      ],
```

```suggestion
      "outFiles": [
        "${workspaceFolder}/packages/typespec-vscode/dist/**/*.cjs",
      ],
```

i think this only needs this